tests/db.py

```diff
@@ -2,4 +2,4 @@
 
 
 def connect():
-    return _connect(host="db", db="test")
+    return _connect(host="localhost", db="test")
```

Comment:
This would break the docker-compose setup, as the mongo service is running with the 'db' host.

Reply:
Oh yeah, i updated for local tests and forgot to change back :/